src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -12,105 +12,325 @@ Sentry's Performance features are currently in beta. For more details about acce
-- What conditions are causing bottlenecks or latency issues in the application
-- Which endpoints or operations consume the most time
+**Traces**
 
-### Trace Propagation Model
+Traces are not an entity in and of themselves. Rather, a trace is defined as the collection of all transactions that share a `trace_id` value. 
 
-Let's refer back to our earlier example of the web application consisting of these high-level components:
+**Transactions**
 
-- Frontend (Single-Page Application)
-- Backend
-- Background Queue
-- Notification Job
+Transactions share most of their properties (start and end time, tags, and so forth) with their root spans, so the same options described below for spans are available in transactions, and setting them in either place is equivalent.
 
-A trace [propagates]({%- link _documentation/performance/performance-glossary.md -%}#propagation) first from the frontend, then to the backend, and later to the background queue or notification job. Collected [spans]({%- link _documentation/performance/performance-glossary.md -%}#span) from each component are sent back to Sentry asynchronously and independently. Instrumented spans received from one component aren't forwarded to the next component. If a span appears to be missing a parent span, it could be an [**orphan span**]({%- link _documentation/performance/performance-glossary.md -%}#orphan-spans).
+Transactions also have one additional property not included in spans, called `transaction_name`, which is used in the UI to identify the transaction. Common examples of `transaction_name` values include endpoint paths (like `/store/checkout/` or `api/v2/users/&lt;user_id&gt;/`) for backend request transactions, task names (like `data.cleanup.delete_inactive_users`) for cron job transactions, and URLs (like `https://docs.sentry.io/performance/distributed-tracing/`) for page-load transactions.
 
-## Transactions
+**Spans**
 
-[{% asset performance/anatomy-of-transaction.png alt="Diagram illustrating how a transaction is composed of many spans." %}]({% asset performance/anatomy-of-transaction.png @path %})
+The majority of the data in a transaction resides in the individual spans the transaction contains. Span data includes:
 
-Traces in Sentry are segmented into [spans]({%- link _documentation/performance/performance-glossary.md -%}#span) called [**transactions**]({%- link _documentation/performance/performance-glossary.md -%}#transaction). When instrumenting the application with tracing, collected spans are grouped within an encompassing top-level span called the **transaction span**. This notion of a transaction is specific to Sentry.
+- `parent_span_id`: ties the span to its parent span
+- `op`: short string identifying the type of operation the span is measuring
+- `start_timestamp`: when the span was opened
+- `end_timestamp`: when the span was closed
+- `description`: longer description of the span's operation, often specific to that instance (optional)
```

Comment:
Per the discussion in #discuss-backend, I think this should be changed to NOT be specific to an instance of a span.

"Longer description of the span's operation, that uniquely identifies the span and is consistent across instances of the span."

Reply:
Hmmm - maybe we mean different things by "specific to the instance," or maybe we're not following that advice ourselves.

Here's a transaction in our org:

![image](https://user-images.githubusercontent.com/14812505/81462595-8f002880-9168-11ea-84df-97e3ab186da4.png)

The transaction name definitely does what you're saying, in that only that endpoint has that transaction name ("uniquely identifies") but every time someone hits that endpoint, the transaction has the same name set ("consistent across instances").

But here's a span in that transaction:

![image](https://user-images.githubusercontent.com/14812505/81576562-4067af80-935d-11ea-8a94-6b0b4deb65df.png)

By my reading, this _isn't_ "consistent across instances," in that it's got the specific link in there, rather than something like `link https://s1.sentry-cdn.com/_static/<hash>/sentry/dist/sentry.css` or even `link https://<cdn host>/_static/<hash>/sentry/dist/sentry.css` (in case we ever end up distributing CDNs around the world).

Before I change any language in the doc, I first want to make sure we're talking about the same thing.